use-cases/client-server-msgs-perf/svmsg_file_server.c

```diff
@@ -296,33 +319,147 @@ main(int argc, char *argv[])
     printf("Start Server, using clock '%s'"
            ": Initiate L3-%slogging to log-file '%s'"
            ", using %s encoding scheme.\n",
-           clock_name(clock_id), l3_log_mode, logfile, loc_scheme);
+           clock_name(Clock_id), l3_log_mode, logfile, loc_scheme);
 
     snprintf(run_descr, sizeof(run_descr), "L3-%slogging %s",
              l3_log_mode, loc_scheme);
 
 #else // L3_ENABLED
 
-    printf("Start Server, using clock ID=%d '%s': No logging.\n",
-           clock_id, clock_name(clock_id));
+    printf("Start Server, using clock ID=%d '%s': No logging"
+           ", %d server-threads, server-msgq-ID=%d.\n",
+           Clock_id, clock_name(Clock_id), num_threads, serverId);
     snprintf(run_descr, sizeof(run_descr), "Baseline - No logging");
 
 #endif // L3_ENABLED
 
-    /* Read requests, handle each in a separate child process */
-    Client_info *clientp = NULL;
-    requestMsg req;
-    responseMsg resp;
+    // Invoke pthread_create() to create n-threads ...
+    svr_thread_config   svr_config = {
+                            .server_id = serverId
+                        };
+    pthread_t   thread_ids[num_threads];
 
     struct timespec ts0 = {0};
-    struct timespec ts1 = {0};
 
+    if (svr_start_threads(thread_ids, num_threads, (void *) &svr_config)) {
```

Comment:
Currently, this will start just 1 server-thread, essentially the same code as before, but serialized through one thread.